### PR TITLE
docs - REPEATABLE READ xact mode is supported. 

### PR DESCRIPTION
**Pull request by litzell**
SERIALIZABLE falls back to REPEATABLE READ.

URLs (larger changes) for review:

- http://docs-litzell-gpdb6.cfapps.io/600/admin_guide/dml.html#topic7
- http://docs-litzell-gpdb6.cfapps.io/600/admin_guide/intro/about_mvcc.html#topic_kl4_pl2_2s__section_f4m_n5n_fs
- http://docs-litzell-gpdb6.cfapps.io/600/ref_guide/sql_commands/BEGIN.html
- http://docs-litzell-gpdb6.cfapps.io/600/ref_guide/sql_commands/SET_TRANSACTION.html
- http://docs-litzell-gpdb6.cfapps.io/600/ref_guide/sql_commands/START_TRANSACTION.html

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/intro/about_mvcc.xml [hunk 1]
```diff
@@ -156,39 +157,36 @@
...
-        Applications that execute transactions in <codeph>SERIALIZABLE</codeph> mode must be
-        prepared to handle transactions that fail due to serialization errors. If
-          <codeph>SERIALIZABLE</codeph> isolation mode is not required by the application, it is
-        better to use <codeph>READ COMMITTED</codeph> mode. </p>
-      <p>The SQL standard specifies that concurrent serializable transactions produce the same
-        database state they would produce if executed sequentially. The MVCC snapshot isolation
-        model prevents dirty reads, non-repeatable reads, and phantom reads without expensive
-        locking, but there are other interactions that can occur between some
+      <p>Greenplum Database treats <codeph>READ UNCOMMITTED</codeph> the same as <codeph>READ
+          COMMITTED</codeph> and <codeph>SERIALIZABLE</codeph> the same as <codeph>REPEATABLE
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/intro/about_mvcc.xml]: READ COMMITTED and SERIALIZABLE -> READ COMMITTED, and treats SERIALIZABLE

File: gpdb-doc/dita/ref_guide/sql_commands/SET_TRANSACTION.xml [hunk 2]
```diff
@@ -25,15 +25,15 @@ SET SESSION CHARACTERISTICS AS TRANSACTION <varname>transaction_mode</varname>
...
+          only see rows committed before the first query or data-modification statement executed in
+          the transaction.</li>
       </ul>
       <p>The SQL standard defines two additional levels, <codeph>READ UNCOMMITTED</codeph> and
-          <codeph>REPEATABLE READ</codeph>. In Greenplum Database <codeph>READ UNCOMMITTED</codeph>
-        is treated as <codeph>READ COMMITTED</codeph>. <codeph>REPEATABLE READ</codeph> is not
-        supported; use <codeph>SERIALIZABLE</codeph> if <codeph>REPEATABLE READ</codeph> behavior is
-        required. </p>
+          <codeph>SERIALIZABLE</codeph>. In Greenplum Database <codeph>READ UNCOMMITTED</codeph> is
+        treated as <codeph>READ COMMITTED</codeph>. <codeph>SERIALIZABLE</codeph> is not supported;
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/SET_TRANSACTION.xml]: Is it correct to say SERIALIZABLE "is not supported"?  The rest of the edits suggest that REPEATABLE READ is the equivalent.

File: gpdb-doc/dita/admin_guide/intro/about_mvcc.xml [hunk 3]
```diff
@@ -111,8 +111,9 @@
           <li><i>Phantom read</i> – a query executed twice in the same transaction can return two
             different sets of rows because another concurrent transaction added rows.</li>
         </ul></p>
-      <p>The SQL standard defines four transaction isolation modes that database systems must
-        support: </p>
+      <p>The SQL standard defines four transaction isolation levels that database systems can
+        support, with the phenomena that are allowed when transactions execute concurrently for each
+        level. </p>
       <table frame="all" id="table_gxb_qsz_nt">
         <title>Transaction Isolation Modes</title>
```
Comment (mkiyama) [inline gpdb-doc/dita/admin_guide/intro/about_mvcc.xml]: Might want to change title to "SQL Transaction Isolation Mode" or something to distinguish from GPDB modes.

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 4]
```diff
@@ -1681,7 +1681,8 @@
           </thead>
```
Comment (mkiyama) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: might want to mention in the GUC description that GPDB treats READ UNCOMMITTED the same as READ COMMITTED and treats SERIALIZABLE the same as REPEATABLE READ

**Pull request merged**